packages/flutter/lib/src/widgets/form.dart

```diff
@@ -651,14 +648,7 @@ class FormFieldState<T> extends State<FormField<T>> with RestorationMixin {
   @override
   void initState() {
     super.initState();
-    if (widget.forceErrorText != null) {
-      // ignore: always_specify_types
-      Future.microtask(() {
-        setState(() {
-          _errorText.value = widget.forceErrorText;
-        });
-      });
-    }
+    _errorText = RestorableStringN(widget.forceErrorText);
```

Comment:
I removed the microtask in favor of a late variable to ensure that the `_errorText` Restorable String is registered in initState with the provided `forceErrorText` instead of registering it with `null` and then use setState to update it.